test/units/TEST-72-SYSUPDATE.sh

```diff
@@ -227,15 +239,35 @@ EOF
     test ! -f "$WORKDIR/xbootldr/EFI/Linux/uki_v1.efi.extra.d/extra.addon.efi"
     test ! -d "$WORKDIR/xbootldr/EFI/Linux/uki_v1.efi.extra.d"
 
-    # Create fourth version, and update through a file:// URL. This should be
+    # Create fourth version, update using updatectl and verify it replaced the
+    # second version
+    new_version "$sector_size" v4
+    systemctl start systemd-sysupdated
+    "$SYSUPDATE" --verify=no check-new
+    updatectl check
```

Comment:
Still not checking for anything. This won't cause the test to fail if an update isn't available.

IMO, it makes sense for `updatectl check` to "fail" if no updates are available. However, we can also add a flag to make the test return success/failure via exit status.

CC @poettering , thoughts?

Reply:
I left this here as a smoke test of the `check` verb rather than using it to actually check that the update is there.  I figured it was better to have it have a new version to check for than putting it with the rest of the tests below.  I should probably add a comment to that effect.

Reply:
i'd always be defensive. i.e. if in doubt then don#t consider something an error if we don#t have to.

(but it's fine to add a switch to make things an error on request)